modules/geo-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -0,0 +1,89 @@
+import {GLTFMaterialParser} from '@luma.gl/experimental';
+import {Model, pbr} from '@luma.gl/core';
+import {SimpleMeshLayer} from '@deck.gl/mesh-layers';
+
+import vs from './mesh-layer-vertex.glsl';
+import fs from './mesh-layer-fragment.glsl';
+
+function validateGeometryAttributes(attributes) {
+  const hasColorAttribute = attributes.COLOR_0 || attributes.colors;
+  if (!hasColorAttribute) {
+    attributes.colors = {constant: true, value: new Float32Array([1, 1, 1])};
+  }
+}
+
+const defaultProps = {
+  // PBR material object. _lighting must be pbr for this to work
+  pbrMaterial: {type: 'object', value: null}
+};
+
+export default class _MeshLayer extends SimpleMeshLayer {
```

Comment:
We can drop the underscore here, and add a JSDoc comment to the class stating that this is an internal layer. If we export it from the module, we can export it as `export {default as _MeshLayer} from '...'`